src/registration/utils.py

```diff
@@ -20,6 +18,12 @@ def get_site(request):
     not installed.
 
     """
+    from django.contrib.sites.models import Site
+    try:
+        from django.contrib.sites.models import RequestSite
+    except ImportError:
+        from django.contrib.sites.requests import RequestSite
+
     if Site._meta.installed:
```

Comment:
Because of https://github.com/lambdalisue/django-inspectional-registration/blob/develop/tests/settings.py#L94